docs/guides/info_plan.md

```diff
@@ -24,7 +34,36 @@ Heavy Hippo     | hippo-1,3,5   | dedicated | 1,3,5
 Loud Lion       | lion-1,3,5    | dedicated | 1,3,5
 Raging Rhino    | rhino-1       | dedicated | 1
 
+```shell
+# Filter out available plans for RabbitMQ
+curl -u :xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxxx \
+  https://customer.cloudamqp.com/api/plans?backend=rabbitmq
```

Comment:
Each of these examples with filtered requests would require: https://github.com/84codes/customer-console/pull/164

Or if we should just reference our docs.cloudamqp.com.

Reply:
Yes we should document it there and reference it